build-docker.sh

```diff
@@ -1,27 +1,36 @@
 #!/usr/bin/env bash
-set -e
+set -e -o pipefail
 
-CONTAINER_NAME=trezor-firmware-env.nix
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+CONTAINER_NAME=${CONTAINER_NAME:-trezor-firmware-env.nix}
+ALPINE_VERSION=${ALPINE_VERSION:-3.12.0}
+ALPINE_RELEASE=${ALPINE_RELEASE:-3.12}
+ALPINE_ARCH=${ALPINE_ARCH:-x86_64}
+ALPINE_CDN=${ALPINE_CDN:-http://dl-cdn.alpinelinux.org/alpine}
```

Comment:
Please order this in the same order as they are used in the `CONTAINER_FS_URL` below